src/pages/forgotPassword.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { ReactElement } from 'react';
+import { NextPageContext } from 'next';
+import { parseCookies } from 'nookies';
+import { ErrorInfo } from '../types';
+import { BaseLayout } from '../layout/Layout';
+import ErrorSummary from '../components/ErrorSummary';
+import FormElementWrapper from '../components/FormElementWrapper';
+import { FORGOT_PASSWORD_COOKIE } from '../constants';
+
+const title = 'Forgot Password - Fares data build tool';
+const description = 'Forgot Password page of the Fares data build tool';
+const id = 'email';
+
+interface ForgotEmailProps {
+    email: string;
+    errors: ErrorInfo[];
+}
+
+const ForgotPassword = ({ email, errors = [] }: ForgotEmailProps): ReactElement => {
+    return (
```

Comment:
Since the return is the only statement in the body we can use the shorthand here:

```
const ForgotPassword = ({ email, errors = [] }: ForgotEmailProps): ReactElement => (
<BaseLayout>
...
</BaseLayout>
)
```